packages/babel-preset-env/src/built-in-definitions.js

```diff
@@ -1,140 +1,259 @@
+const ArrayNatureIterators = [
+  "es.array.iterator",
+  "web.dom-collections.iterator",
```

Comment:
I assume this correctly bails out if not on a browser?

Would be nice to be able to omit it if `targets` did not include a browser, but the <1KB you'd save is probably not worth the effort.

Reply:
I hope that it should be removed for a non-browser target, otherwise we should add logic for removing it.